packages/core/src/render3/instructions/shared.ts

```diff
@@ -2004,12 +2004,12 @@ export function storePropertyBindingMetadata(
 
 export const CLEAN_PROMISE = _CLEAN_PROMISE;
 
-export function getLCleanup(view: LView): any[] {
+export function getOrCreateLCleanup(view: LView): any[] {
```

Comment:
Just a minor suggestion to make this function consistent with the `getOrCreateTViewCleanup` one:
```suggestion
export function getOrCreateLViewCleanup(lView: LView): any[] {
```

Reply:
done.